libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -137,8 +143,25 @@ export function newBucket(
                 } as one of the items for bucket ${localId}.` +
                     "Please make sure that you are not trying to put an array of items into a bucket that only accepts single item.",
             );
+        } else if (typeof i === "object") {
+            if (Object.keys(i).indexOf("Default") > -1) {
+                invariant(
+                    false,
+                    `${contentErrorMessage} It looks like you meant to use of the following display forms: ${Object.keys(
+                        i,
+                    )}.`,
+                );
+            }
+            const keys = intersection(AGGREGATION_KEYS, Object.keys(i));
+            if (!isEmpty(keys)) {
+                invariant(
+                    false,
+                    `${contentErrorMessage} It looks like you meant to use of the following measures: ${keys}.`,
+                );
+            }
+            invariant(false, `${contentErrorMessage} Got unknown content object.`);
         } else {
-            invariant(false, `Contents of a bucket must be either attribute, measure or total. Got: ${i}`);
+            invariant(false, `${contentErrorMessage} Got ${typeof i} unsupported content.`);
```

Comment:
Idea: Should we also stringify the provided invalid content (so the issue is more clear for the user) ?